packages/office-ui-fabric-react/gulpfile.js

```diff
@@ -20,7 +20,7 @@ let {
 } = buildConfig;
 
 let visualTestClean = build.subTask('visualTestClean', (gulp, options, done) => {
```

Comment:
This is just a temporary cleanup. We'll move this over to the clean config in the future.